style.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: #f7f7f9;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+.nav-bar {
+  background-color: #f7f7f9;
+  padding: 20px 15px;
```

Comment:
**_[OPTIONAL] -_** Please let's change the background color of this section from `#f7f7f9` to `white` as to match the template. As we must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

![image](https://user-images.githubusercontent.com/78251927/211937594-65259448-5113-4378-897b-af3d23a94ec9.png)

Figma 👇 (`background-color: #fff`)

![image](https://user-images.githubusercontent.com/78251927/211937224-a1e30050-b8a3-4293-bfbd-1e24ee9ebab3.png)